README.md

```diff
@@ -34,8 +34,9 @@ repositories {
 Add the following dependencies to your `build.gradle` (in your app module) in the `dependencies` block:
 
 ```gradle
-implementation 'net.skyscanner.backpack:backpack-android:36.0.0' // for Android View system
-implementation 'net.skyscanner.backpack:backpack-compose:0.2.0' // for Compose (beta)
+bpkVersion = "36.0.0"
+implementation "net.skyscanner.backpack:backpack-android:bpkVersion" // for Android View system
+implementation "net.skyscanner.backpack:backpack-compose:bpkVersion" // for Compose
```

Comment:
removed the beta while i was at it - we can wait until the announcement was made until merging this and then do a release to make sure the version is correct